doctr/models/_utils.py

```diff
@@ -87,9 +89,9 @@ def extract_rcrops(img: np.ndarray, boxes: np.ndarray) -> List[np.ndarray]:
 
 
 def rotate_page(
-    image: np.ndarray,
-    angle: float = 0.,
-    min_angle: float = 1.
+        image: np.ndarray,
+        angle: float = 0.,
+        min_angle: float = 1.
```

Comment:
Would you mind reverting that change? It will throw linter check errors

Reply:
ok will do